types/core/rolls/dicePool.d.ts

```diff
@@ -38,82 +81,82 @@ declare class DicePool {
     kl: 'keep';
   };
 
+  /* -------------------------------------------- */
```

Comment:
I think these kind of comments are just visual noise
```suggestion
 
```

Reply:
I agree, but that is the same thing as keeping the original foundry.js comments formatting. It helps to spot differences.